src/components/pieCharts/ScoreCardGenEfficiencyDoughnut.js

```diff
@@ -10,8 +10,10 @@ const ScoreCardGenEfficiencyDoughnut = ({ data, uiSettings, peakData }) => {
 
   const colorAndMessage = getGeneratorSizeMessage(usage);
 
-  const percentageUsage = (peakData?.peak / gen_size) * 100;
-  const roundedUsage = percentageUsage.toFixed(2);
+  // const percentageUsage = (peakData?.peak / gen_size) * 100;
+  const percentageUsage = usage;
+  // const roundedUsage = percentageUsage.toFixed(2);
+  const roundedUsage = percentageUsage
```

Comment:
weldone @wazkayd , However I think there is no need declaring another variable hold usage i.e  const percentageUsage = usage; you can pass usage directly where it is needed

Reply:
its now resolved
you may merge please
thanks